src/inputs/select/select.jsx

```diff
@@ -851,9 +851,6 @@ const Select = React.forwardRef(function Select(props, ref) {
                 <div
                     aria-selected={isSelected}
                     className={optionClass}
-                    isDisabled={o.disabled}
-                    isFocused={isFocused}
-                    isSelected={isSelected}
```

Comment:
Thank you!!!!! 👍 🎉 
These console warnings have been driving me _nuts!_ =)

Reply:
Me too.